### PR TITLE
LIIKUNTA-59 | Search area

### PR DESCRIPTION
**Pull request by tuovinensanttu**
As discussed with Tuomas, I'll push just the layout for now so he can proceed with the unified search integration.

### CODE REVIEW

File: src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        <TextInput
+          placeholder="Kirjoita hakusana, esim. uimahalli tai jooga"
+          id="text"
+          name="text"
+          className={styles.inputWithIcon}
+        >
+          <IconSearch aria-hidden="true" />
+        </TextInput>
+        <Select
+          label=""
```
Comment (Jokinen) [inline src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx:36]: Missing label. There should be a prop for visually hiding the label.

File: src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          id="place"
+          placeholder="Valitse paikka"
+          options={mockCategory}
+        />
+        <Select
+          label=""
```
Comment (Jokinen) [inline src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx:42]: Missing label. There should be a prop for visually hiding the label.

File: src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          id="sport"
+          placeholder="Valitse liikuntamuoto/laji"
+          options={mockSportTypes}
+        />
+        <Select label="" placeholder="Valitse alue" options={mockArea} />
+        <Select label="" placeholder="Valitse hinta" options={mockPrice} />
```
Comment (Jokinen) [inline src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx:48]: Missing labels.

File: src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+        <Checkbox id="Box " label="Sopiva liikuntarajoitteisille" />
+        <Checkbox id="Box " label="Lähimmät paikat ensin" />
+        <Checkbox id="Box " label="Näytä vain ilmaiset" />
+        <Checkbox id="Box " label="Avoinna nyt" />
```
Comment (Jokinen) [inline src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx:53]: IDs should be unique within the page. I.e. you should be able to find only a single element with a given ID.

Consider whether these checkboxes should be grouped with `SelectionGroup`.

File: src/pages/search/index.tsx [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import Page from "../../components/page/Page";
+import SearchPageSearchForm from "../../components/search/searchPageSearchForm/SearchPageSearchForm";
+export default function Search() {
```
Comment (Jokinen) [inline src/pages/search/index.tsx:3]: Missing whitespace.

File: src/pages/search/index.tsx [hunk 6]
```diff
@@ -0,0 +1,9 @@
...
+  return (
+    <Page title="Search" description="Search">
+      <SearchPageSearchForm />
+    </Page>
+  );
+}
```
Comment (Jokinen) [inline src/pages/search/index.tsx:9]: Missing `getStaticProps` export.

**Review by Jokinen — APPROVED** [review]
Not sure what level of finish this PR targets, but here are some code style comments I have so far.

File: src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx (hunk 1 above)
Comment (tuovinensanttu) [reply src/components/search/searchPageSearchForm/SearchPageSearchForm.tsx:36]: Hmm, seems that the `hideLabel` prop is only for TextInput 🤔

**Pull request merged**